src/test/resources/pages/instructorFeedbackResultsAjaxByQuestionViewForHelperTwo.html

```diff
@@ -315,16 +315,16 @@
             <tbody>
               <tr>
                 <td class="middlealign">
-                  Team 1
+                  Team 1</td></div>'"
```

Comment:
Just to verify, to see if this is correctly handled, it's based on the indentation level right?